service_client.go

```diff
@@ -114,6 +114,8 @@ func (client *ServiceClient) setMicroversionHeader(opts *RequestOpts) {
 		opts.MoreHeaders["X-OpenStack-Nova-API-Version"] = client.Microversion
 	case "sharev2":
 		opts.MoreHeaders["X-OpenStack-Manila-API-Version"] = client.Microversion
+	case "volume":
+		opts.MoreHeaders["X-OpenStack-Volume-API-Version"] = client.Microversion
```

Comment:
@jtopjian yes, you're right, this piece of code will never be reached if the string is empty. I've overlooked the condition. I'm sorry for that.

However, how does CinderV3 behave in case the `X-OpenStack-Volume-API-Version` is not present? Are such packets rejected?